### DISCUSSION TITLE
Feature request: add link to directory the same way you can add link to a specific file

### DISCUSSION THREAD

**Discussion by mstyp** [gsantner/markor#1690] [Ideas] [answered]
I have a big foulder with documents I want to link to. Its annoying to have to make links for every individual file in the foulder, when I should just be able to, within the document, click a link to open the directory (either in markor itself or with simple file manager app)

unsure if this is feature request or support help, since to me this seems like a bit of a no brainer feature that if you can link to specific files you should also be able to link to specific foulders but for the life of me I cant figure it out. 

there is no option in the UI that I can see when making a link (by clicking paperclip) tried makeing file:/// links like i do for hytml and firefox, tried making intent:/// links like indicated here: https://stackoverflow.com/questions/50111170/android-url-scheme-to-open-file-from-browser-link but nothing seems to work, so i am assuming its a feature not yet added.

saw that making shortcuts on homescreen to specific foulders feature was recently added, but I cant figure out how to get that to work either so I am assuming its just not been implimented in the fdroid app yet.

**Comment by gsantner** [comment, marked as answer]
Link to folders should work since #967 . So update to 2.8.6 if you haven't(via F-Droid).
